keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2641,42 +2771,36 @@ when the feature is disabled.
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
+Workloads not using ResourceClaims should not be impacted because the new code
+will not do anything besides checking the Pod for ResourceClaims.
 
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+When kube-controller-manager fails to create ResourceClaims from
+ResourceClaimTemplates, those Pods will not get scheduled. Bugs in
+kube-scheduler might lead to not scheduling Pods that could run or worse,
+schedule Pods that should not run. Those then will get stuck on a node where
+kubelet will refuse to start them. None of these scenarios affect already
+running workloads.
+
+Failures in kubelet might affect running workloads, but only if containers for
+those workloads need to be restarted.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
+One indicator are unexpected restarts of the cluster control plane
+components. Another are an increase in the number of pods that fail to
+start. In both cases further analysis of logs and pod events is needed to
+determine whether errors are related to this feature.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-<!--
-Describe manual testing that was done and the outcomes.
-Longer term, we may want to require automated upgrade/rollback tests, but we
-are missing a bunch of machinery and tooling and can't do that now.
--->
+This will be done manually before transition to beta by bringing up a KinD
+cluster with kubeadm and changing the feature gate for individual components.
```

Comment:
This isn't just enablement/disablement, but upgrade/rollback. So, it needs to get to the conversions and whether things roundtrip properly, etc.

Reply:
Roundtripping is covered for all types by existing, generic tests. I added a comment about that.